tests/auth/registration.spec.js

```diff
@@ -47,25 +41,18 @@ test.describe("Auth1", ()=>{
 
 
 
-test.describe("Auth2", ()=>{
+test.describe("Auth", ()=>{
     test("The alert message should be visible if user enters invalid name, shorter than 2 simbols", async ({page})=>{
         const name = 'E'
         const lastName = 'Klyemenko'
         
         await page.goto('/')
         
-        const logInButton = page.locator('//div/button[2]')
-        await expect(logInButton, "Sign in button should be visible").toBeVisible()
-        await expect(logInButton, "Sign in button should be enabled").toBeEnabled()
+        const signUpButton = page.locator('div [class="hero-descriptor_btn btn btn-primary"]')
```

Comment:
please use dot syntax for CSS selectors by class
https://developer.mozilla.org/en-US/docs/Web/CSS/Class_selectors

Reply:
please fix selectors in all your tests